src/pal/pal_openbsd.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#if defined(__OpenBSD__) && !defined(_KERNEL)
+#  include "../ds/bits.h"
+#  include "../mem/allocconfig.h"
+#  include "pal_bsd.h"
+
+#  include <stdio.h>
+#  include <string.h>
+#  include <sys/mman.h>
+
+namespace snmalloc
+{
+  class PALOBSD : public PALBSD
+  {
+  public:
+    static constexpr uint64_t pal_features = LazyCommit;
```

Comment:
If you include this line in the PALBSD, then we can remove this file, and just use PALBSD for OpenBSD and extend it for FreeBSD.

Reply:
Yes I considered this but was not sure if one day OpenBSD would need a particular treatment (ie having special mmap flags nothing else has).

Reply:
I think the right thing to do here (from the perspective of making the code readable) is for the BSD PAL to set LazyCommit and for the OpenBSD PAL to declare `pal_features` but set it to `PALBSD::pal_features`, so if anyone needs to extend this class with non-generic behaviour later then they have a reminder to do so.  I'll make that change when I do some cleanup.

Reply:
No problem at all with that, result is the same just make the "inheritance" reminder clearer indeed.